active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,225 @@
+
+![PipsPager, horizontal, visible buttons](images/PipsPager_HorizontalVisible.jpg)
+
+The new PipsPager control can  be used when the page content is not explicitly ordered and a symbol or glyph can instead represent the page number. 
+
+# Description
+Represents a control that provides a interaction and navigation of a paginated layout view (ListView, GridView, ItemsRepeater, DataGrid, etc.) when the page numbers do not need to be visually known.
+
+The PipsPager, like the other pager controls, is independent of the layout views and does not control any of the data being displayed in the layout view.
+
+## Is this the right control?
+Use a PipsPager when the content in the layout is not explicitly ordered by relevancy or you desire a glyph-based representation of numbered pages. This UI is commonly used in photo viewers, app lists, and when display space is limited. The PipsPager can be oriented vertically or horizontally.
+
+### What is a pip?
+Pips represent a unit of numerical value, typically represented as dots, however can be customized to other glyphs like dashes or squares. In the PipsPager control by default, a solid dot will represent each page of the layout. They can be selected by the user to navigate to a specific page.
+
+# Examples
+
+## Create a PipsPager
+A PipsPager with five pages that can be selected by the user. 
+By default, the previous and next buttons are collapsed and the pips are oriented horizontally.
+
+![A sketch of five horizontal dots with the first selected.](images/PipsPager_HorizontalCollapsed.jpg)
+
+XAML
+```XAML 
+    <muxc:PipsPager NumberOfPages="5"/>
+```
+
+## PipsPager with visible buttons
+The visibility of the previous and next buttons can be set via the PreviousButtonVisibility and NextButtonVisibility properties. 
+These buttons allow the user to select a page. The possible values are:
+- Visible: The button is visible and selectable. An exception to this is that if the selected page is on the associated "edge" of the button then the button will be hidden.
+  - The edge for the previous button is the first page and the edge for the next button is the last page.
+  - Note: Hidden means that the button is invisible but does take up layout space.
+- VisibleOnHover: The behavior is the same as Visible *except* that the button is only Visible when the user is hovering over the paging UI with their cursor.
+- Collapsed: The button is not visible to the user and does **not** take up layout space.
+
+![A sketch of five horizontal dots with the first selected and the previous and next buttons are visible](images/PipsPager_HorizontalVisible.jpg)
+
+XAML
+```XAML 
+    <muxc:PipsPager NumberOfPages="5"
+      PreviousButtonVisibility=Visible
+      NextButtonVisibility=Visible />
+```
+
+## Vertical PipsPager with VisibleOnHover button visibility
```

Comment:
I don't think we're perfectly consistent about it, but `Hover` means that the pointer is over something for a while, for example the tooltip case. (The amount of time you need to delay is specified in [UISettings.MouseHoverTime](https://docs.microsoft.com/uwp/api/Windows.UI.ViewManagement.UISettings.MouseHoverTime).)  And `Over` means the pointer is over something, no delay.

In Xaml, setting a button's mode to [ClickMode.Hover](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ClickMode#fields) means the button will click if you put the pointer over it and wait a moment. But [ButtonBase.IsPointerOver](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Primitives.ButtonBase.IsPointerOver) is true the whole time.